common/persistence/sql/sqlplugin/mysql/tls.go

```diff
@@ -42,13 +41,8 @@ func registerTLSConfig(cfg *config.SQL) error {
 		return nil
 	}
 
-	host, _, err := net.SplitHostPort(cfg.ConnectAddr)
-	if err != nil {
-		return fmt.Errorf("error in host port from ConnectAddr: %v", err)
-	}
-
 	// TODO: create a way to set MinVersion and CipherSuites via cfg.
-	tlsConfig := auth.NewTLSConfigForServer(host)
```

Comment:
this is a bug on MySQL

Reply:
because NewTLSConfigForServer is "for connecting as a TLS client to a server with a name we know and may want to verify"? because when i read the function name it makes me think its a config for a server listening on TLS. but this is not an issue with this PR and warrants a bigger discussion about how we are talking about TLS connections.